### PR TITLE
delete idea

### PR DESCRIPTION
**Pull request by Asiasama710**

### CODE REVIEW

**Review by Saif-Alhaider — APPROVED** [review]
i approve this

File: src/main/kotlin/file.kt [hunk 1]
```diff
@@ -2,13 +2,6 @@ fun main() {
 
 }
 
-//fun getTipPercentage(): Int {
-//    return when {
-//        names.size > 5 -> 20
-//        names.size > 0 -> 10
-//        else -> 0
-//    }
-//}
```
Comment (devnoorm7md) [inline src/main/kotlin/file.kt:11]: hi I am Noor

File: src/main/kotlin/file.kt [hunk 2]
```diff
@@ -2,13 +2,6 @@ fun main() {
 
 }
 
-//fun getTipPercentage(): Int {
```
Comment (Taha-cs) [inline src/main/kotlin/file.kt:5]: Don't delete it

**Review by devnoorm7md — APPROVED** [review]

**Review by Taha-cs — APPROVED** [review]

File: src/main/kotlin/file.kt [hunk 3]
```diff
@@ -2,13 +2,6 @@ fun main() {
 
 }
```
Comment (HamSaAli) [inline src/main/kotlin/file.kt:3]: hiii

**Review by HamSaAli — APPROVED** [review]

**Review by Kareem-Shahtout — APPROVED** [review]
Ok!!

**Comment by amna97alaa** [comment]
hiiiii

**Pull request merged**